Gruntfile.js

```diff
@@ -1,5 +1,11 @@
 module.exports = function(grunt) {
 
+    // Configuration options for Amazon S3 service
+    var awsConfig = {
```

Comment:
What if you move all S3 related code down to the same code block? Would be nice to keep it in one place. Or could these vars be moved to secrets?